x/evm/types/params.go

```diff
@@ -1,39 +1,24 @@
 package types
 
-import (
-	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
-	"gopkg.in/yaml.v2"
-)
-
-var _ paramtypes.ParamSet = (*Params)(nil)
-
 // ParamKeyTable the param key table for launch module
-func ParamKeyTable() paramtypes.KeyTable {
-	return paramtypes.NewKeyTable().RegisterParamSet(&Params{})
+func ParamKeyTable() KeyTable {
+	return KeyTable{}
 }
 
 // NewParams creates a new Params instance
 func NewParams() Params {
 	return Params{}
 }
 
-// DefaultParams returns a default set of parameters
-func DefaultParams() Params {
-	return NewParams()
-}
-
 // ParamSetPairs get the params.ParamSet
-func (p *Params) ParamSetPairs() paramtypes.ParamSetPairs {
-	return paramtypes.ParamSetPairs{}
+func (p *Params) ParamSetPairs() ParamSetPairs {
+	return ParamSetPairs{}
```

Comment:
revert this change

Reply:
previously it was using `paramtypes` from cosmos-sdk `x/params` and this folder is not being used anywhere in 0.50.1 so the required types are moved to local param types and `x/params` paramtypes is removed